### DISCUSSION TITLE
`iree-import-tf`: How to get human-readable MLIR output?

### DISCUSSION THREAD

**Discussion by mablinov** [iree-org/iree#18408] [Q&A] [answered]
Hi all,

Is there an equivalent to the `llvm-dis` program, to turn `.mlir` files generated from TensorFlow SavedModels into human-readable `.mlir` files?

**Comment by ScottTodd** [comment, marked as answer]
The `.mlir` files generated there are in the MLIR bytecode format: https://mlir.llvm.org/docs/BytecodeFormat/. Looks like we still aren't setting the file extension to `.mlirbc`.

To convert to MLIR text format, either of these should work:
```
iree-ir-tool cp model.mlirbc -o model.mlir
iree-opt model.mlirbc > model.mlir
```

We should also update our documentation to include those.
